### ISSUE TITLE
Verbose not showing after calling Start-AzureRMAutomationBook

### ISSUE THREAD

**Issue by hvvong** [Azure/azure-powershell#4483]
### Cmdlet(s)
Start-AzureRmAutomationBook

$result= Start-AzureRmAutomationRunbook -ResourceGroupName $rG -AutomationAccountName $Account -Name 'UpdateScript' -Parameters $params -RunOn $HybridRunbookWorker -wait

### Description
I noticed that the latest release fixed a verbose issue when using Get-AzureRmAutomationVariable and Get-AzureRmAutomationJob.  I am running into a similar issue when I use Start-AzureRmAutomationBook.  Once this cmdlet is called, none of my verbose afterwards will show but the script does continue to run.

Here is what we have tried:
- Added _$VerbosePreference = 'Continue'_

- Set  the cmdlet to a variable 
- Added _-wait_ to wait for the other script to complete before continuing on
- Upgraded Azure Powershell to the latest release

Any suggestions or ideas on how to fix this?

**Comment by cormacpayne** [comment]
@vrdmr Hey Varad, would you mind taking a look at this issue?

**Comment by hvvong** [comment]
Hi!  Any update on this?

**Comment by hvvong** [comment]
Hi!  Can someone provide an update on this?  Is there a solution or a workaround?  This is holding up my deployment to production.

**Comment by vrdmr** [comment]
Hi @hvvong, Apologies for the delay in the response.

We have picked this up and should be fixed in the next release. Just to understand the issue a little better, the `-Verbose` flag does not output anything? Not even the following output which contains the JobId?

```

PS C:\Users\vameru> Start-AzureRmAutomationRunbook -RunbookName "AzureAutomationTutorial" -AutomationAccountName "contoso-test-automation-1" -ResourceGroupName "contoso-dev" -Verbose

ResourceGroupName      : contoso-dev
AutomationAccountName  : contoso-test-automation-1
JobId                  : b1234567-12b0-1234-b12b-1234567f320a
CreationTime           : 8/29/2017 5:36:02 PM -07:00
Status                 : New
StatusDetails          : None
StartTime              : 
EndTime                : 
Exception              : 
LastModifiedTime       : 8/29/2017 5:36:02 PM -07:00
LastStatusModifiedTime : 8/29/2017 5:36:02 PM -07:00
JobParameters          : {}
RunbookName            : AzureAutomationTutorial
HybridWorker           : 
StartedBy              : contoso@microsoft.com
```

**Comment by hvvong** [comment]
Hi Varad.  I do get the details of my jobid as shown in your example.  It is in the log of the job that the verbose statements are missing after the call to automation.  

Here how my flow works:  Runbook A runs and I can see the verbose statements in the log of the job.  In the middle of Runbook A, it does a call to Automation Runbook to kick off Runbook B.  Runbook A appears to be continuing with the remaining script but none of the verbose statements in the remaining part of the script shows up in the log.  Only when I go verify my results do I see that Runbook A did complete successfully.  In the meantime, Runbook B completes successfully also.  While everything seems to be running fine, should something fail that one time, there are no verbose statements to see where it failed or why it failed.  I hope that makes sense.  I am still learning all of this so if there is another solution, please let me know.

**Comment by vrdmr** [comment]
From what I we could see, this issue was fixed in the newest release (3.3.1). Could you please update the module and try again?

```
PS C:\Users\vameru> (Get-Module AzureRm.Automation).Version

Major  Minor  Build  Revision
-----  -----  -----  --------
3      3      1      -1     
```

**Comment by vrdmr** [comment]
Resolved with updating the module to 3.3.1 from 3.1.0.
